playground/backend/internal/code_processing/code_processing.go

```diff
@@ -78,7 +94,15 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	prepareFunc := executor.Prepare()
 	go prepareFunc(successChannel, errorChannel)
 
-	if err = processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, nil, nil, errorChannel, pb.Status_STATUS_PREPARATION_ERROR, pb.Status_STATUS_COMPILING); err != nil {
+	ok, err = processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel)
```

Comment:
can you add some comments before each process step function so readers know which step is being processed?

Reply:
Yes, I will.